templates.dist/estate/default_detail.php

```diff
@@ -330,10 +330,11 @@ function renderEnergyCertificate(string $energyCertificateType, array $energyCla
 					</div>
 				</div>
 			<?php } ?>
-
-			<div class="oo-units">
-				<?php echo $pEstates->getEstateUnits(); ?>
-			</div>
+			<?php if (!empty($pEstates->getEstateUnits())) : ?>
+				<div class="oo-units">
+					<?php echo $pEstates->getEstateUnits(); ?>
+				</div>
+			<?php endif; ?>
```

Comment:
```suggestion
			<?php if (!empty($pEstates->getEstateUnits())) : ?>
				<?php echo $pEstates->getEstateUnits(); ?>
			<?php endif; ?>
```
Ich würde das div auch in die Datei der Einheitenlisten packen. Es macht eigentlich keinen Sinn das es hier drin ist und so wäre alles zu den Units in einer Datei. Was meinst du?